--- a/build/config/linux/dri/BUILD.gn
+++ b/build/config/linux/dri/BUILD.gn
@@ -8,14 +8,8 @@
 
 pkg_config("dri") {
   packages = [ "dri" ]
-  dri_driver_dir = exec_script(pkg_config_script,
-                               common_pkg_config_args + pkg_config_args + [
-                                     "--dridriverdir",
-                                     "dri",
-                                   ],
-                               "string")
-  defines = [ "DRI_DRIVER_DIR=\"$dri_driver_dir\"" ]
+  # Hardcode DRI driver directory to bypass pkg-config
+  defines = [ "DRI_DRIVER_DIR=\"/usr/lib/x86_64-linux-gnu/dri\"" ]
 }
